### ISSUE TITLE
Regression with New-HPOVServerProfileAttachVolume results in exception

### ISSUE THREAD

**Issue by ChrisLynchHPE** [HewlettPackard/POSH-HPOneView#314]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
```PowerShell
$vol = Get-HPOVStorageVolume -Name "My Test Volume"
$ServerProfile = Get-HPOVServerProfile -Name "My Server Profile"
New-HPOVServerProfileAttachVolume -ServerProfile $ServerProfile -Volume $vol
```

### Actual Behavior
Exception generated:

```PowerShell
New-HPOVServerProfileAttachVolume : The requested resource '/rest/server-profiles/available-storage-systems?enclosureGr
oupUri=/rest/enclosure-groups/642c8446-023b-4040-a92e-98659806c49e&serverHardwareTypeUri=/rest/server-hardware-types/CF
6310CF-8ADF-4341-9844-EDBAA08984A2' could not be found. Verify that the profile exists and try again.
At line:1 char:1
+ New-HPOVServerProfileAttachVolume -ServerProfile $sp -Volume $vol1 -v ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (URI:String) [New-HPOVServerProfileAttachVolume], ResourceNotFoundExcept
   ion
    + FullyQualifiedErrorId : ResourceNotFound,New-HPOVServerProfileAttachVolume
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`):
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  
Output from `$PSVersionTable` on your Windows Host:

```PowerShell
# Output from $PSVersionTable
Name                           Value
----                           -----
PSVersion                      5.1.16299.248
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.16299.248
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by ChrisLynchHPE** [comment]
Similar to #312

**Comment by ChrisLynchHPE** [comment]
This will be fixed in the next update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 4.0.1612.2800](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.0.1612.2800) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.400/4.0.1612.2800).

Closing issue.
